spec/beaker/host/cisco_spec.rb

```diff
@@ -14,10 +14,10 @@ module Cisco
 
     describe '#prepend_commands' do
 
-      context 'for cisco-5' do
+      context 'for cisco-nexus' do
 
         before :each do
-          @platform = 'cisco-5-x86'
+          @platform = 'cisco-nexus-x86'
```

Comment:
We're going with cisco_nexus-MAJOR_VERSION-x86_64 & cisco_ios_xr-MAJOR_VERSION-x86_64.  Which is currently cisco_nexus-7-x86_64 & cisco_ios_xr-6-x86_64